scripts/release-next-major.js

```diff
@@ -149,6 +149,7 @@ let currentGitHash = null
     await promiseSpawn(`yarn`, [`bootstrap`], {
       shell: true,
```

Comment:
https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options

> shell <boolean> | <string> If true, runs command inside of a shell. Uses '/bin/sh' on Unix, and process.env.ComSpec on Windows. A different shell can be specified as a string. See Shell requirements and Default Windows shell. Default: false (no shell).

When removing this the behavior didn't change. Was there a specific reason for `shell`?

Reply:
It's necessary for my windows installation, if I don't add it then it can't find yarn